src/App.test.jsx

```diff
@@ -2,11 +2,16 @@ import React from 'react';
 import Enzyme, { mount } from 'enzyme';
 import EnzymeAdapter from 'enzyme-adapter-react-16';
 import { Provider } from 'react-redux';
+import firebase from 'firebase/app';
 import { findByTestAttr } from './utilities/testutils';
 import App from './App';
 import store from './Redux/store';
-import LoginChange from './Redux/actions/LoginChange';
+import UserChange from './Redux/actions/UserChange';
+import auth from './utilities/mockAuth';
+import firestore from './utilities/mockFirestore';
 
+firebase.firestore = firestore;
+firebase.auth = auth;
```

Comment:
So you can mock firestore from your utility files to prevent having to repeat this every file. Just inject a mock `jest.mock('firebase');` then override the return value.

More details: https://jestjs.io/docs/en/mock-functions.html#mocking-modules